### DISCUSSION TITLE
create an empty model repository

### DISCUSSION THREAD

**Discussion by ekaratza** [Open-Systems-Pharmacology/Forum#1863] [Models]
Hello, 
can you please create an empty model repository for me (Eleni Karatza).
Please find the requested info below:
- name of your model : Valproic Acid (if it exists then VPA)
- short description of the model repository: Whole-body PBPK model of valproic acid including non-linear binding with albumin in adults, children and children with hypoalbuminemia. 
- repository topics: pbpk, pediatrics, non-linear binding
We want to upload the mode described in this publication: 
https://pubmed.ncbi.nlm.nih.gov/39298079/

Thank you, 
Eleni

**Comment by Yuri05** [comment]
Created: https://github.com/Open-Systems-Pharmacology/Valproic-Acid
